nvflare/app_common/workflows/scatter_and_gather.py

```diff
@@ -229,9 +230,17 @@ def control_flow(self, abort_signal: Signal, fl_ctx: FLContext) -> None:
                 data_shareable.set_header(AppConstants.NUM_ROUNDS, self._num_rounds)
                 data_shareable.add_cookie(AppConstants.CONTRIBUTION_ROUND, self._current_round)
 
+                operator = {
+                    TaskOperatorKey.OP_ID: self.train_task_name,
+                    TaskOperatorKey.METHOD: OperatorMethod.BROADCAST,
+                    TaskOperatorKey.TIMEOUT: self._train_timeout,
+                    TaskOperatorKey.AGGREGATOR: self.aggregator_id,
+                }
+
                 train_task = Task(
                     name=self.train_task_name,
                     data=data_shareable,
+                    operator=operator,
```

Comment:
what does this mean to have "operator" defined inside "scatter_and_gather" workflow?

Does it mean that this "scatter_and_gather" class support multiple hierarchical SAG?

What if the T1 aggregator wants to have different argument/parameters VS T2 aggregator?